### PR TITLE
es6-destructuring | Danny Andreev

### PR DESCRIPTION
**Pull request by dannyandreev**
image

### CODE REVIEW

**Review by scbowler — CHANGES_REQUESTED** [review]
Looks like there is stuff missing, I can only see the first object `book1`

**Comment by dannyandreev** [comment]
image

File: es6-destructuring/index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+</head>
+<body>
+  <script>
+    const book1 = {
+      title: 'Goodnight Punpun',
+      author: 'Inio Asano',
+      libraryID: 3353
+    }
+
+    let  { title, author, libraryID } = book1
```
Comment (scbowler) [inline es6-destructuring/index.html:17]: Since the values won't be changed you want to use `const` here

File: es6-destructuring/index.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    console.log("The title of the book is "+title+", the author is "+author+", and the library id is " + libraryID)
+
+
+    const book2 = {
+      title: "Les Fleurs du mal",
+      author: "Charles Baudelaire",
+      libraryID: 2345
+    }
+
+    let { title2, author2, libraryID2 } = book2
```
Comment (scbowler) [inline es6-destructuring/index.html:28]: `const`

File: es6-destructuring/index.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        libraryID: 3245
+      },
+      {
+        title: "Dead Dead Demon's De De De De Destruction",
+        author: 'Inio Asano',
+        libraryID: 1233
+      }
+    ];
+
+    let [book3, book4, book5] = library
```
Comment (scbowler) [inline es6-destructuring/index.html:56]: `const`

File: es6-destructuring/index.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    console.log(book3)
+    console.log(book4)
+    console.log(book5)
+    let [, , , book6] = library
```
Comment (scbowler) [inline es6-destructuring/index.html:60]: `const`

**Review by scbowler — CHANGES_REQUESTED** [review]
A few changes needed, see below comments

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**